style.css

```diff
@@ -0,0 +1,275 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Poppins;
+  font-size: 10px;
```

Comment:
The font of the elements in the header looks a bit small and disproportionate. Also, in general it's a good practice not to use a smaller font than 16px on a website as it's not readable. 
The problem that will come up after you make that change it that everything else will get very big, since you made it relative to the current html font size by using "em" units..